### PR TITLE
Adventurer/Towner Changes

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request

**Tl;dr** - I went ahead and did some stat changes and trait changes, trying to buff under-utilized roles that were left behind while trying to make classes less.. meta-choices and instead more character building and play style choice.

Changes are a lot to list, but a brief?
- **Clerics** lost heavy armor training, war cleric got medium armor and dodge expert. Cleric has good stats, so this is honestly fine - it's no longer a Watchman with less skills slightly but holy powers. Also, Templars are limited to T1 spells, so are Paladins. Clerics deserve more than them, due to lower skill, but less than Acolyte. So, they have level 2 cap on spells. (And I gave them a shirt.)
- **Amazon** lost their redundant probability bikini code, instead it's just flat set that way now. Also, stopped them from having duped shoes; you get 50-50 chance at gladiator or leather. Untouched otherwise; just QOL.
- **Bard** lost their medium armor trait. But gave them guaranteed 1 medicine and 5 skill in music; plus they still have dodge expert.
- **Dwarf Vagrant** is basically unchanged, just clarified it's a dwarf unique roll to know when you roll it, plus gave 1 extra unarmed.
- **Paladin** is mostly untouched; gave it 5 slots, 2 more than Templar, since it's supposed to be a 'rare roll'. (It's not rare, just now limited.) Also, gave them 3 climbing instead of random chance between 2-3.
- **Dwarf Ranger** got dwarf added to it's name so it's clear it's unique instead of being a dwarf and seeing 2 ranger classes. Buffed it's unarmed by 1, climbing by 1, and crossbow by 1. Makes it compete with the other ranger class.
- **Mage adventurer** got an extra bit of polearms, bow, guaranteed level 1 wrestling, unarmed, and climbing. Same with swords, knives, crafting, and medicine. Plus level 2 riding. Also toned down their stat-gain from being old, no longer legendary; instead master.
- **Ranger** no longer gets.. two cloaks that override eachother, it's 50-50 which color you get. Also, gave extra 1 to sword skill (now 3), made bows set to master instead of toss up between 4-6 skill (all it does is effect draw time anyway), and gave them 3 knives instead of 2. This way competes with Rogue a bit better.
- **Necromancer** I've been told a mix of things this roll is playable, then it isn't, etc. So - I've just given it a limit of 1 due to how powerful it is. Will change if needed, maybe make it 2 - but heard like 2 horror story rounds of Necromancers existing and idk if that was admin done or rando players. They're decently strong given their unique shit and immune to zombies.
- **Rogue** - ok absolutely shit-tons of changes, I nerfed stuff like stealing (so no longer instant steal stuff), buffed some stuff like bows, gave them lock-picking, and tried to make them different from sub-classes and ranger. Took away medium armor, but gave them a bow and they kept dodge expert. (You are a stealthy thief stealing TVs from flats, not an armed bank robber with a list of felonies)
- **Sorceress** Basically same kinda changes as mage, gave some guaranteed skills. Made their default arcane skill 3 instead of 2, but now from being old you don't get 3 you get 2. Adds up to 5 if old.
- **Warrior** no longer gets heavy armor skill no matter what, you get medium armor now. BUT the monster hunter subclass gets heavy armor skill, regular warrior instead gets better stats and good medium armor instead. Also, gave monster hunter shield skill of 1 vs regular warrior of 2. Also - took the Battle Axe due to 12 strength min, which made some unable to use, gave a steel axe and wood shield. Should be a fair trade; warrior player feedback seemed to agree.
- **Fisher** gave slight buffs by making it so you don't have random pick chance of stuff, but also removed some like sword skill chance of 0-1 (with 0 more likely) because - why? Given them guaranteed axes though. Also, it looks like I changed their fishing skill - I didn't if you look, I just made it actually standardized.
- **Hunter** got given a guaranteed of 1 sword skill instead of random chance pick, same with bows and crossbow. Soft buff, no armor training or dodge expert though so still civi-alligned but capable. 

**Non-Adventurer stuff**
I said I'd fix mercenaries if issues arose or feedback, so doing my part.
- **Desert Rider** got dodge expert since they start with light armors. 
- **Steppesman** was underpowered like thought, but better to be underpowered than over. Gave them crit resistance, like Barb, since they have light armor already.

## Why It's Good For The Game

I can write an entire manifesto here but I think limiting adventurer slots is bad. Instead making them play into certain play styles, by encouraging you to play certain ways while allowing flexibility is great. Support roles support, dps roles have good skills, tanks still have some armor training. Etc.

I am looking for staff, coder, and player feedback since - I fully admit, I have not played every role here. I've played hunter often in the past elsewhere, rogue once or twice, ranger a few times, warrior prob once, etc. But, I'm not immune to opinionated bias so - best to get feedback.

The class hit kinda hardest is Cleric (and I guess Paladin?) - why? Simple, you are an _axuillary_ healer; you have combat stats unlike acolytes. And templars are a limit 3 slot role with a mix. So, if you do want based stats, go templar - or even Paladin. But if you want freedom like a wandering priest, with class options for your play style? Play cleric.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Hey how do you rank up form like E to a SSS rank like in my manga/anime with these changes?

**Comment by Stutternov** [comment]
> Hey how do you rank up form like E to a SSS rank like in my manga/anime with these changes?

Depends on skills; you see, Trilgy, you have to get on that grindset. I, personally, turn on my Andrew Tate inspiration video and grab a shield. Then, I start bashing the dummy on weak intent. When I've hit apprentice, at that point I swap on over to pvp training on weak intent. 

But, some skills? It's like, those mangas without leveling. You gotta just be born like that. You gotta have that power within you. Such as some like, I believe, lockpicking and climbing may also be like that. You just gotta have that dog in you, you know?

**Comment by Stutternov** [comment]
More serious note - I'll touch up towner role skill if input is given. Those I'm DEATHLY afraid of touching since I'll fully admit I've never played them. I know they need a face-lift, but I'm unsure in which ways.

**Comment by YuiY1997** [comment]
The cleric nerf is foul

**Comment by Stutternov** [comment]
> If you're already touching things can you give the desert mercenary dodge expert so they can actually dodge in their gear?

Could add that, will do so now.

> The cleric nerf is foul

These are actually all direct asks by coders and players for the most part; basically only me offering maybe we could make cleric class choices different with the dodge expert v medium armor option.

**Comment by YuiY1997** [comment]
> > If you're already touching things can you give the desert mercenary dodge expert so they can actually dodge in their gear?
> 
> Could add that, will do so now.
> 
> > The cleric nerf is foul
> 
> These are actually all direct asks by coders and players for the most part; basically only me offering maybe we could make cleric class choices different with the dodge expert v medium armor option.

Honestly the spells slot nerf is more than enough. I would give the war clerics heavy armor and give life clerics dodge. IF they can only get up to level 2 anyway that means that incredibly useful spells like the astratans second heal or necra cleric's churn undead are completely off the table for them.

**Comment by Stutternov** [comment]
> Honestly the spells slot nerf is more than enough. I would give the war clerics heavy armor and give life clerics dodge. IF they can only get up to level 2 anyway that means that incredibly useful spells like the astratans second heal or necra cleric's churn undead are completely off the table for them.

The issue is, by giving them heavy armor they are literally Paladin + as they have better spell capabilities and Paladin only has marginally better stats/skills. WAY too many roles have heavy armor training to and extent.

The issue clerics cause is they fully and utterly replace all church jobs in their current state. Something does need to be done regarding their combat prowess and the complete ability to replace all church roles atop of it.

**Comment by YuiY1997** [comment]
> > Honestly the spells slot nerf is more than enough. I would give the war clerics heavy armor and give life clerics dodge. IF they can only get up to level 2 anyway that means that incredibly useful spells like the astratans second heal or necra cleric's churn undead are completely off the table for them.
> 
> The issue is, by giving them heavy armor they are literally Paladin + as they have better spell capabilities and Paladin only has marginally better stats/skills. WAY too many roles have heavy armor training to and extent.
> 
> The issue clerics cause is they fully and utterly replace all church jobs in their current state. Something does need to be done regarding their combat prowess and the complete ability to replace all church roles atop of it.


Then don't nerf their combat ability AND their support ability. IF you nailed one they would be useful enough for the one role they are suited for. I would make it so that life and war cleric is a binary choice between if you want to be a combat role with t1 spells and heavy armor or a support role with t2 or even 3 spells. It would also make life vs war domain more of a meaningful choice.

**Comment by Stutternov** [comment]
> Then don't nerf their combat ability AND their support ability. IF you nailed one they would be useful enough for the one role they are suited for. I would make it so that life and war cleric is a binary choice between if you want to be a combat role with t1 spells and heavy armor or a support role with t2 or even 3 spells. It would also make life vs war domain more of a meaningful choice.

I could remove Paladins then if you want only T1 spells and heavy armor. Because it's the same thing at that point.

Or if you want full spells then I might as well remove accoylte.

This is the issue, they step badly on both classes toes to the point neither are commonly played. This is why Cleric is actually what Level 0 devotion is called in code; it was meant to be a literal low devotion role, not a Acolytes with combat stats to fully replace the need for the church and its roles.

**Comment by YuiY1997** [comment]
> > Then don't nerf their combat ability AND their support ability. IF you nailed one they would be useful enough for the one role they are suited for. I would make it so that life and war cleric is a binary choice between if you want to be a combat role with t1 spells and heavy armor or a support role with t2 or even 3 spells. It would also make life vs war domain more of a meaningful choice.
> 
> I could remove Paladins then if you want only T1 spells and heavy armor. Because it's the same thing at that point.
> 
> Or if you want full spells then I might as well remove accoylte.
> 
> This is the issue, they step badly on both classes toes to the point neither are commonly played. This is why Cleric is actually what Level 0 devotion is called in code; it was meant to be a literal low devotion role, not a Acolytes with combat stats to fully replace the need for the church and its roles.
> 
> What this does is put them as an in-between both. They aren't as good at spells as an Acolyte, who has **malices** to their stats, but they are more useful as support than a Paladin.
> 
> What I could do is toss them dodge expert and medium armor, but I'm personally unwilling to give them heavy armor back and I was directly requested by multiple people to do a stop-gap fix for their spells.


Giving them dodge AND medium I think would make up for how much they have been gutted with this. I think that's a solid compromise position.


> This isn't DnD, no caster should be rocking heavy plate. Clerics are wizards/white mages, they can sit in light armor.

Do you think they got a mace and a strength stat just for the hell of it?

**Comment by Stutternov** [comment]
> Giving them dodge AND medium I think would make up for how much they have been gutted with this. I think that's a solid compromise position.

I'm totally fine with that then, it lets them use medium or light armor (light armor gets dodge bonus from dodge expert) and they can spec as they like. I'll do that then rather choosing which they want; they'll just get both on all sub-class options

**Comment by Stutternov** [comment]
> If you're already touching things can you give the desert mercenary dodge expert so they can actually dodge in their gear?

Also - done. Giving Steppesman crit resistance to give a reason to choose him over desert rider since no medium armor; I think that's really fair given he's a speed build as is.

**Comment by YuiY1997** [comment]
> I'm totally fine with that then, it lets them use medium or light armor (light armor gets dodge bonus from dodge expert) and they can spec as they like. I'll do that then rather choosing which they want; they'll just get both on all sub-class options

goated thanks for listening to the feedback

**Comment by faaaay** [comment]
I spoke more about this in #staff but blah blah blah, this PR looks good, +1 or something.

**Comment by Stutternov** [comment]
> I spoke more about this in #staff but blah blah blah, this PR looks good, +1 or something.

1 reddit gold

**Comment by faaaay** [comment]
> 1 reddit gold

changed my mind. moderators delete this pr and destroy this coder.

**Comment by Troopermoores** [comment]
Man I'll be real paladin ain't that good to be nerfing to three slots lmao, journeyman skills and healing id minimal keep it to 5 if you REALLY just want to look like you're touching every adventurer

**Comment by Troopermoores** [comment]
Or if you're going to treat them like templar's give em Templar stats lmao

**Comment by Stutternov** [comment]
> Man I'll be real paladin ain't that good to be nerfing to three slots lmao,

Slots increased to 5. 3 was at request of others given Templar is limited to 3. 

> Or if you're going to treat them like templar's give em Templar stats lmao

image
